mix.exs

```diff
@@ -43,10 +43,13 @@ defmodule GRPC.Mixfile do
       # This is the same as :gun 2.0.0-rc.2,
       # but we can't depend on an RC for releases
       {:gun, "~> 2.0.1", hex: :grpc_gun},
+      {:mint, "~> 1.4.2"},
       {:cowlib, "~> 2.11"},
       {:protobuf, "~> 0.10", only: [:dev, :test]},
       {:ex_doc, "~> 0.28.0", only: :dev},
-      {:dialyxir, "~> 1.1.0", only: [:dev, :test], runtime: false}
+      {:dialyxir, "~> 1.1.0", only: [:dev, :test], runtime: false},
+      {:ex_machina, "~> 2.7.0", only: :test},
```

Comment:
I'd rather not add ExMachina as a dependency here

Reply:
Out of curiosity, this is just to avoid having the dependency on the repo. Right?